src/networkremote/client.cpp

```diff
@@ -0,0 +1,65 @@
+#include "client.h"
+#include "core/application.h"
+
+NetworkRemoteClient::NetworkRemoteClient(Application *app, QObject *parent)
+  : QObject(parent),
+    app_(app),
+    incomingMsg_(new NetworkRemoteIncomingMsg(this)),
+    outgoingMsg_(new NetworkRemoteOutgoingMsg(app, this))
+{
+  player_ = app_->player();
+}
+
+NetworkRemoteClient::~NetworkRemoteClient()
+{
+}
+
+void NetworkRemoteClient::Init(QTcpSocket *socket)
+{
+  socket_ = socket;
+  QObject::connect(incomingMsg_,&NetworkRemoteIncomingMsg::InMsgParsed,this, &NetworkRemoteClient::ProcessIncoming);
+  incomingMsg_->Init(socket_);
+  outgoingMsg_->Init(socket_, player_);
+}
+
+QTcpSocket* NetworkRemoteClient::GetSocket()
+{
+  return socket_;
+}
+
+void NetworkRemoteClient::ProcessIncoming()
+{
+  switch (incomingMsg_->GetMsgType())
```

Comment:
snake case for variables, not camel case

Reply:
Not sure what's wrong